src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherTaxServiceImplTest.java

```diff
@@ -0,0 +1,302 @@
+		boolean isForeignVendorAndTaxReviewRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewRequired(KFSConstants.PaymentPayeeTypes.VENDOR, PAYMENT_REASON_THAT_REQUIRES_TAX_REVIEW_FOR_FOREIGN_VENDOR, FOREIGN_VENDOR_HEADER_ID);
+        assertTrue("isForeignVendorAndTaxReviewRequired method should have returned true when input is foreign vendor and payment reason that requires tax review", isForeignVendorAndTaxReviewRequired);
+    }
+	
+	@Test
+	public void testIsForeignVendorAndTaxReviewRequired_false_PayeeTypeNotVendor()  {
+		boolean isForeignVendorAndTaxReviewRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewRequired(KFSConstants.PaymentPayeeTypes.EMPLOYEE, PAYMENT_REASON_THAT_REQUIRES_TAX_REVIEW_FOR_FOREIGN_VENDOR, FOREIGN_VENDOR_HEADER_ID);
+        assertFalse("isForeignVendorAndTaxReviewRequired method should have returned false when input payee type is not Vendor", isForeignVendorAndTaxReviewRequired);
+    }	
+	
+	@Test
+	public void testIsForeignVendorAndTaxReviewRequired_false_notForeign()  {
+		boolean isForeignVendorAndTaxReviewRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewRequired(KFSConstants.PaymentPayeeTypes.VENDOR, PAYMENT_REASON_THAT_REQUIRES_TAX_REVIEW_FOR_FOREIGN_VENDOR, NON_FOREIGN_VENDOR_HEADER_ID);
+        assertFalse("isForeignVendorAndTaxReviewRequired method should have returned false when input is non foreign vendor and payment reason tha trequires tax review", isForeignVendorAndTaxReviewRequired);
+    }
+	
+	@Test
+	public void testIsForeignVendorAndTaxReviewRequired_false_notPaymentReasonWithNoTaxReview()  {
+		boolean isForeignVendorAndTaxReviewRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewRequired(KFSConstants.PaymentPayeeTypes.VENDOR, PAYMENT_REASON_THAT_DOES_NOT_REQUIRE_TAX_REVIEW_FOR_FOREIGN_VENDOR, FOREIGN_VENDOR_HEADER_ID);
+        assertFalse("isForeignVendorWithTaxReviewRequired method should have returned false when input is foreign vendor and payment reason that does not require tax review", isForeignVendorAndTaxReviewRequired);
+    }
+	
+	@Test
+	public void testIsForeignVendorAndTaxReviewNotRequired_true(){
+		boolean isForeignVendorAndTaxReviewNotRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewNotRequired(KFSConstants.PaymentPayeeTypes.VENDOR, PAYMENT_REASON_THAT_DOES_NOT_REQUIRE_TAX_REVIEW_FOR_FOREIGN_VENDOR, FOREIGN_VENDOR_HEADER_ID);
+        assertTrue("isForeignVendorAndTaxReviewNotRequired method should have returned true when input is foreign vendor and payment reason that does not require tax review", isForeignVendorAndTaxReviewNotRequired);
+	}
+	
+	@Test
+	public void testIsForeignVendorAndTaxReviewNotRequired_false_PayeeTypeNotVendor(){
+		boolean isForeignVendorAndTaxReviewNotRequired = disbursementVoucherTaxService.isForeignVendorAndTaxReviewNotRequired(KFSConstants.PaymentPayeeTypes.EMPLOYEE, PAYMENT_REASON_THAT_DOES_NOT_REQUIRE_TAX_REVIEW_FOR_FOREIGN_VENDOR, FOREIGN_VENDOR_HEADER_ID);
```

Comment:
Another test where the service is being invoked for a non-vendor payee but with a non-null vendor header ID.